test/verifiedTests/testCalcGroupStats/testCalcGroupStats.m

```diff
@@ -0,0 +1,36 @@
+% The COBRAToolbox: testCalcGroupStats.m
+%
+% Purpose:
+%     - testCalcGroupStats tests the functionality of calcGroupStats()
+%       and checks solution against a known solution.
+%
+% Authors:
+%     - Lemmer El Assal February 2017
+%
+
+% define the path to The COBRAToolbox
+pth = which('initCobraToolbox.m');
+CBTDIR = pth(1:end-(length('initCobraToolbox.m') + 1));
+
+initTest([CBTDIR, filesep, 'test', filesep, 'verifiedTests', filesep, 'testTools']);
+
+% load reference data - too complex to embed
+load ref.mat;
+
+data = ones(5)*6;
```

Comment:
The style guide mention "Use spaces around operators, e.g., `i + 1` instead of `i+1`". Furthermore, It is clearer to write `data = 6 * ones(5);`  than `data = ones(5)*6;`.